generator/src/main/java/org/corfudb/generator/operations/NestedTxOperation.java

```diff
@@ -0,0 +1,55 @@
+package org.corfudb.generator.operations;
+
+import org.corfudb.generator.State;
+import org.corfudb.runtime.exceptions.TransactionAbortedException;
+
+import java.util.List;
+
+/**
+ * Created by rmichoud on 7/26/17.
+ */
+public class NestedTxOperation extends Operation {
+
+    private final int maxNest = 20;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)